tests/pauli/dla/test_lie_closure.py

```diff
@@ -678,3 +678,67 @@ def test_non_hermitian_error(
         ops = [np.array([[0.0, 1.0], [0.0, 0.0]])]
         with pytest.raises(ValueError, match="At least one basis matrix"):
             _ = qml.lie_closure(ops, matrix=True)
+
+
+X0 = qml.matrix(X(0))
+Y0 = qml.matrix(Y(0))
+Z0 = qml.matrix(Z(0))
+
+
+@pytest.mark.all_interfaces
+class TestLieClosureInterfaces:
+    """Test input for matrix inputs from AD interfaces"""
```

Comment:
Do we need an Autograd test?

Reply:
added autograd tests for lie_closure and trace_inner_product

Reply:
Nice thanks!
Yeah I think the plan is to move away from it being default/required, but definitely still supported afaik.